### PR TITLE
feat: added Jotai screen and added Onyx content

### PR DESCRIPTION
**Pull request by gedu**

### CODE REVIEW

File: package-lock.json
Comment (adhorodyski) [inline package-lock.json:1]: i think lets stick to the yarn lock, will be easier to resolve conflicts in the future

File: src/pages/jotai-onyx/JotaiOnyxScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import {ONYX_KEYS} from '../../lib/onyx-keys';
+
+type JotaiOnyxScreenProps = {
```
Comment (adhorodyski) [inline src/pages/jotai-onyx/JotaiOnyxScreen.tsx]: just `Props` would do, it's only used underneath and scoped to a component

File: src/pages/jotai-onyx/JotaiOnyxScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import {ONYX_KEYS} from '../../lib/onyx-keys';
+
+type Props = {
+  betas: string[];
+};
+
+function JotaiOnyxScreen({betas}: Props) {
+  return (
+    <View>
+      {betas ? betas.map(beta => <Text>{beta}</Text>) : <Text>No betas</Text>}
```
Comment (adhorodyski) [inline src/pages/jotai-onyx/JotaiOnyxScreen.tsx:13]: need to check for `betas.length > 0`, `[]` will fail now

**Review by adhorodyski — APPROVED** [review]

File: src/pages/jotai-onyx/JotaiOnyxScreen.tsx (hunk 2 above)
Comment (gedu) [reply src/pages/jotai-onyx/JotaiOnyxScreen.tsx:13]: just placeholder code

**Pull request merged**